_internals/configuration.md

```diff
@@ -118,329 +118,329 @@ When `ANSI_DEFAULTS` is enabled, it enables the following ISO settings:
 ODBC and OLE DB automatically set `ANSI_DEFAULTS` to `ON` when connecting.
 The driver and provider then set `CURSOR_CLOSE_ON_COMMIT` and
 `IMPLICIT_TRANSACTIONS` to `OFF`. So `CURSOR_CLOSE_ON_COMMIT` is actually `OFF` at
-connect time, but `ANSI_DEFAULTS` is `ON`. 
+connect time, but `ANSI_DEFAULTS` is `ON`.
 
-### babelfishpg_tsql.ansi_null_dflt_off
+### `babelfishpg_tsql.ansi_null_dflt_off`
 
-Modifies the behavior of the session to override the default nullability of new 
+Modifies the behavior of the session to override the default nullability of new
 columns when the ANSI `NULL` default option for the database is on.
 
-`ON` setting is not allowed for option `ANSI_NULL_DFLT_OFF`. 
+The setting `ON` is not allowed for option `ANSI_NULL_DFLT_OFF`.
 Please use `babelfishpg_tsql.escape_hatch_session_settings` to ignore this
 variable.
 
-### babelfishpg_tsql.ansi_null_dflt_on
+### `babelfishpg_tsql.ansi_null_dflt_on`
 
-Modifies the behavior of the session to override default nullability of new columns 
+Modifies the behavior of the session to override default nullability of new columns
 when the ANSI `NULL` default option for the database is `false`.
-`OFF` setting is not allowed for the option `ANSI_NULL_DFLT_ON`. Please use 
+Setting to `OFF` is not allowed for the option `ANSI_NULL_DFLT_ON`. Please use
 `babelfishpg_tsql.escape_hatch_session_settings` to ignore resulting errors.
 
 
-### babelfishpg_tsql.ansi_nulls
+### `babelfishpg_tsql.ansi_nulls`
 
-Specifies ISO compliant behavior of the "equal" (`=`) and "not equal" (`<>`) comparison 
+Specifies ISO compliant behavior of the "equal" (`=`) and "not equal" (`<>`) comparison
 operators when they are used with `NULL` values.
 
 
-### babelfishpg_tsql.ansi_padding
+### `babelfishpg_tsql.ansi_padding`
 
-`OFF` setting is not allowed for this option. Please use 
+Setting to `OFF` is not allowed for this option. Please use
 `babelfishpg_tsql.escape_hatch_session_settings` to ignore errors.
 
 
-### babelfishpg_tsql.ansi_warnings
+### `babelfishpg_tsql.ansi_warnings`
 
-`OFF` setting is not allowed for this option. Please use 
+Setting to `OFF` is not allowed for this option. Please use
 `babelfishpg_tsql.escape_hatch_session_settings` to ignore errors.
 
 
-### babelfishpg_tsql.arithabort
+### `babelfishpg_tsql.arithabort`
 
-`OFF` setting is not allowed for this option. Please use 
+Setting to `OFF` is not allowed for this option. Please use
 `babelfishpg_tsql.escape_hatch_session_settings` to ignore errors.
 
 
-### babelfishpg_tsql.arithignore
+### `babelfishpg_tsql.arithignore`
 
-`ON` setting is not allowed for this option. Please use 
+The setting `ON` is not allowed for this option. Please use
 `babelfishpg_tsql.escape_hatch_session_settings` to ignore errors.
 
 
-### babelfishpg_tsql.concat_null_yields_null
+### `babelfishpg_tsql.concat_null_yields_null`
 
 If enabled, concatenating a `NULL` value produces a `NULL` result.
```

Comment:
Why is there an explanation for some cases (like this one) but not for others? For example, arothabort and arithignore are far from trivial. Either include an explanation everywhere or remove it everywhere.

Reply:
Please tell me the meaning of these options, and I will dutifully add it.